### PR TITLE
Updating interop protos to be consistent with Java

### PR DESCRIPTION
**Pull request by nmittler**

### CODE REVIEW

File: test/cpp/interop/messages.proto [hunk 1]
```diff
@@ -85,6 +87,10 @@ message SimpleResponse {
   optional string oauth_scope = 3;
 }
 
+message SimpleContext {
+  optional string value = 1;
```
Comment (tbetbetbe) [inline test/cpp/interop/messages.proto]: N.B. In proto3, there's no need for the optional keyword.

**Comment by ctiller** [comment]
What's the plan for keeping these in one place?

**Comment by nathanielmanistaatgoogle** [comment]
#526 is an open issue in which we can plan to have a plan; there's one suggestion in there now but not much development.

**Comment by ctiller** [comment]
Is this dead?

**Comment by ctiller** [comment]
@ejona86 - do you know the status of this?

**Comment by ejona86** [comment]
I think Java is still pretty close to what is in grpc repo. We mainly changed things like "syntax3" and "java_package".

File: test/cpp/interop/messages.proto (hunk 1 above)
Comment (ejona86) [inline test/cpp/interop/messages.proto]: At this point it is just being consistent with the rest of the file though.

**Comment by ejona86** [comment]
@ctiller, the status of what? This PR or keeping all the protos in one place?

I keep pushing back on keeping all the protos in one place, because I don't think it will work as well as people think it would work, and when I explain the issues we will encounter it seems they understand why it won't work like they're thinking, but then I hear discussions on putting them in one place again.

**Comment by ctiller** [comment]
I was mostly curious about this PR. Trying to decruft a little, and
something outstanding from Feb seems like cruft :)

**Comment by ejona86** [comment]
I think the status has been the same since it was created: "someone should review it."

**Comment by nathanielmanistaatgoogle** [comment]
@ejona86: please write down your pushback in #526? I don't think I've yet heard why it won't work as well as I'd like to think it would work.

**Comment by ctiller** [comment]
Closing stale PR.

**Pull request closed without merging**